lib/scripts/diffExperimentBuilds.py

```diff
@@ -0,0 +1,169 @@
+import argparse
+import os
+import subprocess
+import sys
+import shutil
+
+import benchpark.paths
+
+sys.path.append(str(benchpark.paths.benchpark_home) + "/spack/lib/spack")
+import llnl.util.tty.color as color  # noqa: E402
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description="""Compare YAMLs generated from 'system init' between two different commits of Benchpark.
+        (check if the output of system init was changed between commits).
+        
+        Usage: benchpark-python diffExperimentBuilds.py""",
+        formatter_class=argparse.RawTextHelpFormatter,
+    )
+    parser.add_argument(
+        "-o",
+        "--old",
+        type=str,
+        default="develop",  # develop from 3/4/25
+        help="Commit hash or branch name for the old version of Benchpark (default: develop).",
+    )
+    parser.add_argument(
+        "-n",
+        "--new",
+        type=str,
+        default="pearce8-patch-3",
```

Comment:
Should this be "develop"?

Reply:
I'm using this right now to compare develop and this branch. So changing this to develop would not be of use. If it becomes CI, It should be whatever the target commit is against develop.

Reply:
It is the `--new` parameter